lnbits/wallets/lndrest.py

```diff
@@ -114,7 +114,12 @@ async def create_invoice(
                 hashlib.sha256(unhashed_description).digest()
             ).decode("ascii")
 
-        r = await self.client.post(url="/v1/invoices", json=data)
+        try:
+            r = await self.client.post(url="/v1/invoices", json=data)
+        except Exception as exc:
```

Comment:
- the error should also be logged. We cannot rely on the catcher that will log it (maybe it just displays on the UI)